docs/upgrade.html

```diff
@@ -159,6 +159,9 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
                         </li>
                         <li>The deprecated <code>sendOffsetsToTransaction(Map&lt;TopicPartition, OffsetAndMetadata&gt;, String)</code> method has been removed from the Producer API.
                         </li>
+                        <li>The <code>flush</code> method now detects potential deadlocks and logs a warning when used inside a callback.
```

Comment:
this is unnecessary as it is just a log.

Reply:
Personally, I tend the other way on this. If someone is now getting errors logged, this explains why. No big deal either way, I suppose. Just my 2 cents.